tests/integration/nems_subscription_create/test_lambda_handler.py

```diff
@@ -17,7 +24,7 @@
     ],
 )
 @pytest.mark.vcr("casette.yml")
-def test_valid_patient_data(patient_data_file_path: str):
+def test_valid_patient_data(patient_data_file_path: str, mocker: MockFixture):
```

Comment:
The assert line here isn't strong enough - please can we make it assert statusCode==201 AND that there is a valid UUID in the header?